lua/nvim-treesitter/parsers.lua

```diff
@@ -938,6 +938,18 @@ list.lalrpop = {
   maintainers = { "@traxys" },
 }
 
+list.v = {
+  install_info = {
+    url = "https://github.com/vlang/vls",
+    files = { "src/parser.c", "src/scanner.c" },
+    location = "tree-sitter-v/tree_sitter_v",
```

Comment:
yeah, this is quite hacky. Even I forgot that we have to do this hack since it will copy the parser to a `tree-sitter-{lang}` repo :man_facepalming:

Reply:
we could auto-add that `tree-sitter-{lang}/` prefix since its not obvious that it is necessary

Reply:
Oh yah that's odd thing to set see. `tree-sitter-{lang}/` make sense to have it pre-added.

Maybe in the future location would be the name of the directory and clone would check for the location value and moved it, instead of the whole repo, even then it is still hacky 😆.